packages/services/server/src/index.ts

```diff
@@ -413,10 +415,14 @@ export async function main() {
             organizationMembers: new OrganizationMembers(
               storage.pool,
               new OrganizationMemberRoles(storage.pool, logger),
-              storage,
               logger,
             ),
           }),
+        (logger: Logger) =>
+          new OrganizationAccessTokenStrategy({
+            logger,
+            cache: registry.injector.get(OrganizationAccessTokensCache),
+          }),
```

Comment:
This registers authentication and authorization to the GraphQL API using the new organization access tokens.